spec/plane_spec.rb

```diff
@@ -17,14 +17,40 @@
 
 describe Plane do
 
-  xit 'is flying when created'
-
-  xit 'can land'
-
-  xit 'is landed after landing'
-
-  xit 'can take off'
-
-  xit 'is flying after take off'
-
-end
+  subject { Plane.new }
+
+   it 'is flying when created' do
+    expect(subject.flying).to eq true    
+  end
```

Comment:
Lint/BlockAlignment: `end` at 24, 2 is not aligned with `it 'is flying when created' do` at 22, 3